service/dynamodb/expression/condition_test.go

```diff
@@ -207,25 +219,26 @@ func TestCompare(t *testing.T) {
 			},
 		},
 	}
-	for testNumber, c := range cases {
+	for _, c := range cases {
 		exprNode, err := c.lhs.Equal(c.rhs).buildCondition()
```

Comment:
Suggest using `t.Run(c.name, func(t *testing.T) {...` for nested test cases like this.